frontend/src/app.scss

```diff
@@ -1,18 +1,45 @@
 :global(body){
```

Comment:
I replaced all the spaces with tabs in this file. See comment below pointing out the actual changes in the file other than just fixing whitespace